daprdocs/data/components/bindings/cloudflare.yaml

```diff
@@ -0,0 +1,8 @@
+- component: Cloudflare Queues
+  link: cfqueues
```

Comment:
Can we call the referenced file "cloudflare-queues" so that the description is more descriptive and explicit?

Reply:
Done, but I am keeping the "cfqueues" as alias because it's hardcoded in the source:

https://github.com/dapr/components-contrib/blob/5922b602d48967a1452071c3fcb6d976f6b95cf7/bindings/cloudflare/queues/cfqueues.go#L33